### PR TITLE
[ru] finishing translation of profilling guide

### PR DESCRIPTION
**Pull request by kauz**

### CODE REVIEW

**Comment by nschonni** [comment]
Ping @nodejs/nodejs-ru

File: locale/ru/docs/guides/simple-profiling.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ app.get('/newUser', (req, res) => {
 
   username = username.replace(/[!@#$%^&*]/g, '');
 
-  if (!username || !password || users.username) {
+  if (!username || !password || users[username]) {
```
Comment (alexandrtovmach) [inline locale/ru/docs/guides/simple-profiling.md:39]: If it's a typo in original document, you need to create PR with fix, and then translate it in that way

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: locale/ru/docs/guides/simple-profiling.md (hunk 1 above)
Comment (kauz) [reply locale/ru/docs/guides/simple-profiling.md:39]: @alexandrtovmach It's actually already fixed there: https://github.com/nodejs/nodejs.org/commit/435c45612a8d6ae0f07d0dc18670687998e85f72#diff-d717fbc920c1977110094bda1eebfc4b
I better note this in ci message. Next time will do.

Comment (alexandrtovmach) [reply locale/ru/docs/guides/simple-profiling.md:39]: okay, I see

**Comment by alexandrtovmach** [comment]
@kauz Please resolve merge conflicts

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**